iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -332,6 +332,20 @@ dn_pipe_receiver_file_dirs=data/datanode/system/pipe/receiver
 # If its prefix is "/", then the path is absolute. Otherwise, it is relative.
 pipe_consensus_receiver_file_dirs=data/datanode/system/pipe/consensus/receiver
 
+# pipe_consensus_deletion_file_dir
+# If this property is unset, system will save the data in the default relative path directory under the IoTDB folder(i.e., %IOTDB_HOME%/${dn_system_dir}/pipe/consensus/deletion).
+# If it is absolute, system will save the data in the exact location it points to.
+# If it is relative, system will save the data in the relative path directory it indicates under the IoTDB folder.
+# If there are more than one directory, please separate them by commas ",".
```

Comment:
Does this support more than one directory?

Reply:
fixed